drop-storage/src/lib.rs

```diff
@@ -0,0 +1,106 @@
+use std::str::FromStr;
+pub mod error;
+use slog::{debug, Logger};
+use sqlx::{
+    migrate::MigrateDatabase, sqlite::SqliteConnectOptions, ConnectOptions, Sqlite, SqlitePool,
+};
+
+use crate::error::Error;
+
+type Result<T> = std::result::Result<T, Error>;
+// SQLite storage wrapper
+pub struct Storage {
+    _logger: Logger,
+    #[allow(dead_code)]
+    conn: SqlitePool,
+}
+
+#[derive(Debug, strum::Display, strum::EnumString, PartialEq, Eq)]
```

Comment:
`strum` would probably be replaced with something like this https://github.com/launchbadge/sqlx/blob/main/examples/postgres/files/src/main.rs#L4

Reply:
That's a nice suggestion, even though the structure is not really used in this MR and thus I totally removed it